connector/github/github.go

```diff
@@ -34,16 +34,22 @@ type Config struct {
 	ClientID     string `json:"clientID"`
 	ClientSecret string `json:"clientSecret"`
 	RedirectURI  string `json:"redirectURI"`
-	Org          string `json:"org"`
```

Comment:
@estroz btw this behavior _has_ changed. You'll start prefixing "(org):" to any group.